packages/yoroi-ergo-connector/src/inject.js

```diff
@@ -403,11 +400,26 @@ if (shouldInject()) {
  * Returns a PNG base64 encoding of the favicon
  * but returns empty string if no favicon is set for the page
  */
-async function convertImgToBase64(url, outputFormat) {
-    // as a safety precaution, we only load the favicon if it's the same origin as the website
-    // if we don't do this, we might get a CORS error anyway
-    // I don't know if any websites set their favicon to external websites, so it should not be a problem
-    const response = await fetch(url, {mode: 'same-origin'});
+async function convertImgToBase64(origin, urls) {
+    let response;
+    for (url of urls) {
+        try {
+            const mode = url.includes(origin) ? 'same-origin' : 'no-cors';
+            response = await fetch(url, { mode });
+            break;
+        } catch (e) {
+            if (String(e).includes('Failed to fetch')) {
+                console.warn(`[yoroi-connector] Failed to fetch favicon at '${url}'`);
+                continue;
+            }
+            console.error(`[yoroi-connector] Failed to fetch favicon at '${url}'`, e);
+            throw e;
```

Comment:
Are you sure you want to `throw` here?

Reply:
Yeah, I think it might be too dangerous to just continue in this case, because we cannot be sure yet what the actual issue is happening and why. But might be a good idea to try to continue tho. Will think a bit and update the PR if needed 🤔